src/formio.common.ts

```diff
@@ -31,6 +31,36 @@ export interface FormioForm {
   submissionAccess?: AccessSetting[];
 }
 
+export interface GridColumn {
```

Comment:
@vlad-shusterman could you also please make sure that you use this `GridColumn` type everywhere where column / columns are used in angular-formio? 

A good example where this is not done yet is here: https://github.com/formio/angular-formio/blob/refactoring/grids/src/grid/grid.component.ts#L36 

Please find other examples yourself and fix it there.